test/Ocelot.AcceptanceTests/Steps.cs

```diff
@@ -181,12 +184,23 @@ public async Task StartFakeOcelotWithWebSocketsWithConsul()
 
     public void GivenThereIsAConfiguration(FileConfiguration fileConfiguration)
         => GivenThereIsAConfiguration(fileConfiguration, _ocelotConfigFileName);
-
+
+    public async Task GivenThereIsAConfigurationAsync(FileConfiguration fileConfiguration)
+        => await GivenThereIsAConfigurationAsync(fileConfiguration, _ocelotConfigFileName);
```

Comment:
I disagree. Are you sure about it? 

Take a look at the section [Prefer async/await over directly returning Task](https://github.com/davidfowl/AspNetCoreDiagnosticScenarios/blob/master/AsyncGuidance.md#prefer-asyncawait-over-directly-returning-task)

Note: I updated the reference link at the MR's summary because it was the wrong one.